### DISCUSSION TITLE
Getting inactivity time

### DISCUSSION THREAD

**Discussion by kaiserkiwi** [NativePHP/laravel#220] [Ideas]
If you know the macOS app for the time tracking tool toggle track, then you probably know this feature. This app can check how long a user was inactive on their Mac. Not how long the app was inactive but instead how long there were no user inputs whatsoever. (Mouse or Keyboard)

This was useful if I had to leave my workplace to help some coworker or if I started a learning video and forgot to change the time entry. Just so you get an idea why this could be useful.

I would love to see something like that possible for NativePHP. Probably only possible with user consent but this ist probably for the best.

**Comment by simonhamp** [comment]
@kaiserkiwi any of the OS-specific functionality is going to be abstracted away by the runtime (Electron/Tauri) - so it would be great if you could do a little digging online to see if there is anything out there to support this and come back with some notes as this will massively aid in any future implementation of such a feature.
